xml/System.Collections.Generic/HashSet`1.xml

```diff
@@ -1389,7 +1391,7 @@
 ## Examples  
```

Comment:
missing the note - can you please review they're not missing in the rest of the changes as well?